web/src/components/CreateTestPlugins/Rest/steps/RequestDetails/BodyField/BodyFieldContainer.tsx

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+import {BodyMode} from './useBodyMode';
+
+export const BodyFieldContainer = styled.div<{bodyMode: BodyMode}>`
```

Comment:
Can we just pass a boolean to this component? so we can do, $isDisplaying, and the logic to match the body type can be done at the parent level